frontend/src/metabase/nav/components/WhatsNewNotification/WhatsNewNotification.unit.spec.tsx

```diff
@@ -0,0 +1,134 @@
+import { VersionInfoRecord } from "metabase-types/api";
+import {
+  createMockVersionInfo,
+  createMockVersionInfoRecord as mockVersion,
+} from "metabase-types/api/mocks";
+import {
+  createMockSettingsState,
+  createMockState,
+} from "metabase-types/store/mocks";
+import { renderWithProviders, screen } from "__support__/ui";
+import * as domUtils from "metabase/lib/dom";
+import { WhatsNewNotification } from "./WhatsNewNotification";
+
+const notification = () => screen.queryByText("See what's new");
+
+const render = ({
+  embedded = false,
+  lastAcknowledged = null,
+  currentVersion = "v0.48.0",
+  versions = [
+    mockVersion({ version: "v0.48.1" }),
+    mockVersion({
+      version: "v0.48.0",
+      releaseNotesUrl: "https://metabase.com/releases/48",
+    }),
+    mockVersion({ version: "v0.47.0" }),
+  ],
+}: {
+  embedded?: boolean;
+  currentVersion?: string;
+  lastAcknowledged?: string | null;
+  versions?: VersionInfoRecord[];
+}) => {
+  jest.spyOn(domUtils, "isWithinIframe").mockReturnValue(embedded);
+
+  const versionMock = { tag: currentVersion };
```

Comment:
We don't create objects for types in `metabase-types` directly. Please use the dedicated `createMock...` helper. If there is none, please create it.